plugin/pkg/admission/imagepolicy/certs_test.go

```diff
@@ -17,195 +17,193 @@ limitations under the License.
-oECNgTvOEFkBP+OFqFGBJZGfY3/J5h0tTy4lLZXaImzzx8sGGNLLc8R+uyTIO3VV
-qIso2uXB+vzPgMrueflt5yp7hoJjI0c+qEktUg5n+WJFAFteI9LCngN+xwRWVEgp
-rjKVPcT9zio8tLJOhcSPA7q6lORUkwbPWHyNDpamvldnqjhgp5Ceq5f/qfoWPzvM
-H5o72Ax2WduxST+P+hCOqZReUmTaGzAKb5rJwdEpmbnDZ3kSR08aT/40m/EG1SvQ
-pi0b3QECgYEA/mRGIjaYPQr+tw3Sz8g76t3PYfrglro60HdLBn2IUpj2sEpazNId
-2aPFPb58whL+VPmUfXbpPH+wW/+wWpRw4MraFkJanbOjDiEGXK5ZoUQIDZJWUSwf
-oCge5uacU69weC67UyPYmK1e+A/gaFw1Dz729jLxtB3rGWKxEGbWEc0CgYEA3eiP
-hv0GxbdEEbSfQoSPKbBHGI9spaqAIcqL+dSsx3m6Ckqx0El/xi9mQkITgqs2gyqI
-o2T/3yDli9oF4+3Plz0wrZ11auOWX+nhKfACtF679I1PL0UOavXF0FVgOfwOIqdG
-jp4QQV7USkbTP9ZOHo90Y8G4rmTEdMZ/VsH490ECgYEA8u/bsiyk8haf7Tx8SAWW
-gtLUi2NEO20ZYZ+qvEYBe6+sVeqMD/HQo9ksMazKA6ST0Z6O2cpHLolaaGEjjz0X
-FvVhk8RGOTglzQZoxvWRjtojPqKzX81dXlsyN5ufSqPOKlemeN1QqW1XtlmjGsaD
-vU2KFs/L1xCDRbjkEx/B6zkCgYBmqeE9InKvpknnpxjHPWy+bL93rWMmgesltv9r
-ZelJoBdiC4yYQGjM18EHhmpgWbWumU79yQxXvnB0czmmaa9Q2Q5cRCy+duxrE1kI
-ffHCYNG0ImwwAlLZSTtrVxRdvy8K+Ti7YoVCuQyeEIZLUmpx2QyP2mAGzrfVDsB6
-8uKsAQKBgQDO+PmADra91NKJP1iVuvOK8iEy/Z14L03uKtF3X9u8vLdzQZa/Q/P9
-hXOX9ovFwSBQOOfgb+/+QRuPL4xxi1J8CFwrSWCEeFgrDijl9DS6aNY6BWHDA8p6
-8V7Adb04cnenj8QjYYN8/mqsQlHSoAIxeAlUoJpq+pk7O8PAfbjgMw==
+MIIEpAIBAAKCAQEAtxfleQUGsX91NgDgRM9RDCGDkzLaeYzyv56FPiQLoC22w9LO
+oPJV8THUbWGte079XHEFDKd9g4wNKBY/iemhSwMCMg5nIEYMpGjHqCaT5Q41zepf
+jRThiax3lOAso4AQfGHnvdx4bnq4n5ykem2bl6EOpRyN9LY81D63m7sJFJeBGW4D
+bnPboiMlVTJAbv4D92WCLjs+J+lgJ9L6COCP+PswmQQDJd/frCoamY1e+NN12Lpx
+mFnmCrkXEtqM43ReA8Tn2kEO7gmgIccwb75tIVsvPlOao28+VNC/ETPA6FWIP15b
+dBxZZifYnCFtaK7AksaoJuGpeZ6rMCJoDkdJnQIDAQABAoIBAGgvJNiUT5a43E16
+eiMthQTOHTypd0ArQAZhgZg2kpb27ETJs+JJgzXYEPSc3EaKUAlGFoV42Sb9jo78
++fLUCn/QmnM5Vt06fv8e0F2Sdh2k1GDzuKNi+mOIzBLe7QesW7UrUN7skWrsUhog
+3Eb+D8dcUiUJ9o0FFeAAQCJRb31TMNq3S3CaPFbEJaW2d92RqhkRgsTjLrTdHqqg
+++1AIEcyhSvsIZdWYoekNQey8Rtlcnt42FrDH/C1gl7SwICP5sviYmIPJSxjYrU+
+7h4OukYuTgsUs9Hx+dHoNKhhJzUk+8vY9w9Vy2zvDKi6bDFWMVrPH8hFxZ0cozW+
+Nhfp8QECgYEA8Fd/0FhCBDsoFHGszVs3x52i6iyp+rdTSpcEEtsiukec3xKZbl3k
+nIN3LDMJMvOwqukI2fHQeR4H+PeJ5mU7Js5j4k6imo8tUVQKEsGPfthgBXNsHYcm
+h42YOggv+ZqRf2tebs4vUoJCLgNEskuChRo/g7dl6AlvZOOLuz1E4s0CgYEAwwWW
+njt49CUacuCHCAOzSpVEM4RsapGJyS0wgIyVn9KvNcZU9eg+9p3uEsG8Nc6OsY7S
+tYdRzXGqbIb5VeDTbf2aCdNw7tA1FsFtBcHBeiCcyQSFk9sViDVmVMIhdupj8vcP
+QQcK2xoViz2RRHAcJnw7SGkvRPlvxgWf9RLpIhECgYEA4BalPGvd+bXyc8VdegdP
+5dWrna3ROsuNZGGT0cTyBz9YG4Fq/0WaLvhAuDxJmnEuStyGfTcCyrrechrTP7cp
+RNBwvwPUhkly1n6F+eDxWzKBrPNmXc2ydfbmdU1G9fJRML26SZQC+WosLFytw5O1
++UZ06+mmoYeSjj2+/fBtdw0CgYBbukAqspxkcSMsjzDT8zGcTfLjLFD2Gvgs2IVs
+oTMrFiR07ZE+jHs48fVtexQ0VrBFEYFIAprWP7VCEIYr/3BS1qpiHsKKgQHHbzkB
+MS34bGhyIB9X+P76rusvyJOG5B4YxF23X0Ga5nzlnhK6luSWsW4M0/R4w73v7Otc
+kQV4kQKBgQCCLLYPyBOPYVcJXHoCaW+e7zLnozu7G1LzfiHyx5TeJHiZjNEcATV9
+1kEdB5MhE+IPm/23SGQBoe4wVefXi2CcqXkQpQuVovjsKDg9WIaEReJoHmiZtSHr
+O0z0y6y03n9vAZ/C9EZakV4txkgZiZOoI/AbI8QMwaTHwstTUJIkkg==
 -----END RSA PRIVATE KEY-----`)
 
 var clientCert = []byte(`-----BEGIN CERTIFICATE-----
-MIIC+jCCAeKgAwIBAgIJAMvo2rkGpEURMA0GCSqGSIb3DQEBCwUAMCExHzAdBgNV
```

Comment:
Why is this showing a diff?

Reply:
@mikedanese 

the `certs_test.go` file  is generated by the `gencerts.sh` script.

```
// This file was generated using openssl by the gencerts.sh script
// and holds raw certificates for the imagepolicy webhook tests.
```
I changed the script and regenerated the `certs_test.go` file, so the file contents is changed.

```
//lint:file-ignore U1000 Ignore all unused code, it's generated
```

Reply:
I'd recommend reverting the diffs in this file other than the lint comment

Reply:
@liggitt 

thanks for your review. it's reverted now.